x-pack/plugins/observability_solution/observability_ai_assistant_app/public/plugin.tsx

```diff
@@ -103,7 +103,8 @@ export class ObservabilityAIAssistantAppPlugin
     const appService = (this.appService = createAppService({
       pluginsStart,
     }));
-
+    const isEnabled = appService.isEnabled();
+    if (isEnabled){
```

Comment:
I'm surprised there are no whitespace changes 😄

Reply:
Guess my linting hadn't kicked in :confused: